modules/guides/pages/serverless/lambda.adoc

```diff
@@ -141,46 +112,40 @@ output:
 
 === go1.x on x86_64
 
-Grab an archive labelled `benthos-lambda` from the https://github.com/{project-github}/releases[releases page^]
+Grab an archive labelled `redpanda-connect-lambda` from the https://github.com/{project-github}/releases[releases page^]
 page and then create your function:
 
-[source,sh]
+[source,sh, subs="attributes+"]]
 ----
 LAMBDA_ENV=`cat yourconfig.yaml | jq -csR {Variables:{BENTHOS_CONFIG:.}}`
 aws lambda create-function \
   --runtime go1.x \
-  --handler benthos-lambda \
+  --handler redpanda-connect-lambda \
```

Comment:
Does the handler change name? Just getting an AWS account set up so haven't been able to test yet.

Reply:
Yes, that's the correct one (it should match the binary name).